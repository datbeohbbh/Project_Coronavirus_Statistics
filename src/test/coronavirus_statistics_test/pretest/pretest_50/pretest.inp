50 100
q 1 31 152
? 4 1
? 7
q 1 6 187
q 1 18 330
q 2 18 257
? 4 3
? 4 1
q 2 31 5
? 6
q 2 18 35
q 1 14 285
? 4 1
? 7
q 2 6 67
q 1 22 171
q 2 31 106
q 2 18 19
q 3 6 109
q 1 42 380
q 1 20 263
q 2 22 127
q 1 35 267
? 4 7
q 1 47 332
? 4 9
q 1 38 173
? 6
q 1 9 148
q 2 20 131
q 2 14 116
q 1 21 441
q 3 35 152
q 1 43 463
q 2 18 6
? 6
? 4 8
q 2 6 1
q 3 18 10
q 3 35 94
? 4 12
q 3 31 15
q 1 3 208
q 2 6 10
q 2 42 130
q 3 20 47
q 2 21 429
q 3 43 218
q 1 17 384
? 4 8
? 7
q 1 18 346
? 7
q 1 25 444
q 1 10 199
q 1 4 97
q 3 38 143
q 1 6 250
? 4 11
q 3 4 87
q 2 31 21
q 3 38 8
q 2 35 3
? 4 14
? 4 10
? 4 17
? 4 1
q 1 13 249
q 1 43 218
q 1 47 160
? 4 6
? 7
q 1 15 367
q 3 4 8
? 4 1
q 1 27 7
q 2 4 1
q 3 6 228
q 3 42 246
? 6
q 1 41 74
q 2 21 10
? 7
? 7
? 4 9
? 4 9
? 7
? 4 8
? 6
? 4 3
? 4 11
? 4 2
? 7
? 6
? 4 18
? 4 17
? 4 16
? 7
? 4 4
? 7