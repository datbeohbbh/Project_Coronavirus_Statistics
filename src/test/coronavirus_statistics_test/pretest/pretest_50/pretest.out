Query ? 4 1 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 152
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 3 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 187
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 73
-> Has recovered : 257
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 257 people

Query ? 4 1 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 38
-> Has recovered : 292
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 7 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 285
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 380
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 311 people

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 317 people

Query ? 4 8 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 169
-> Has recovered : 116
-> Was dead : 0

Query ? 4 12 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 441
-> Has recovered : 0
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 169
-> Has recovered : 116
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 246 people

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 246 people

Query ? 4 11 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 208
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 250
-> Has recovered : 78
-> Was dead : 109

Query ? 4 10 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 199
-> Has recovered : 0
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 384
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 5
-> Has recovered : 132
-> Was dead : 15

Query ? 4 6 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 44
-> Has recovered : 127
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 246 people

Query ? 4 1 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 2
-> Has recovered : 0
-> Was dead : 95

Query ? 6 :
The city with the highest recovery is city with the index 21
Quantity: 429 people

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 337 people

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 337 people

Query ? 4 9 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 44
-> Has recovered : 127
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 44
-> Has recovered : 127
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 337 people

Query ? 4 8 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 22
-> Has recovered : 0
-> Was dead : 151

Query ? 6 :
The city with the highest recovery is city with the index 21
Quantity: 439 people

Query ? 4 3 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 4
-> Has recovered : 130
-> Was dead : 246

Query ? 4 11 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 85
-> Has recovered : 131
-> Was dead : 47

Query ? 4 2 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 2
-> Has recovered : 439
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 337 people

Query ? 6 :
The city with the highest recovery is city with the index 21
Quantity: 439 people

Query ? 4 18 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 367
-> Has recovered : 0
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 349
-> Has recovered : 317
-> Was dead : 10

Query ? 4 16 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 249
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 337 people

Query ? 4 4 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 5
-> Has recovered : 132
-> Was dead : 15

Query ? 7 :
The city with the highest death is city with the index 6
Quantity: 337 people

